corehq/apps/app_manager/tests/data/days_ago_suite.xml

```diff
@@ -1,220 +1,117 @@
 
-
 <suite version="1" descriptor="Suite File">
-
-
-
```

Comment:
https://github.com/dimagi/commcare-hq/pull/6719/files?w=1 if you don't want to see all this